docs/comparison/nbdb.md

```diff
@@ -22,28 +22,28 @@ kernelspec:
 Requirements:
 
 ```sh
-pip install scikit-learn sklearn-evaluation ploomber jupysql
+pip install scikit-learn sklearn-evaluation ploomber ploomber-engine jupysql nbformat jupytext
```

Comment:
Are you sure jupytext isn't the dependency of one of our projects?

Reply:
I think `nbformat` is a dependency in `ploomber-engine`

Reply:
@gtduncan please remove duplicated dependencies.

Reply:
i can see scikit-learn, ploomber, jupysql etc in sklearn-evaluation's dependencies. Any reason these are added here?